Analysis/Tasks/PWGHF/CMakeLists.txt

```diff
@@ -52,3 +57,8 @@ o2_add_dpl_workflow(hf-task-lc
                     SOURCES taskLc.cxx
                     PUBLIC_LINK_LIBRARIES O2::Framework O2::AnalysisDataModel O2::AnalysisCore O2::DetectorsVertexing
                     COMPONENT_NAME Analysis)
+
+o2_add_dpl_workflow(hf-task-jpsi                                
+                    SOURCES taskJpsiToEE.cxx
```

Comment:
The task can be just taskJpsi.cxx
Here we can keep it general so we add all the other Jpsi decay channels later into the same task